roles/openshift_node/tasks/main.yml

```diff
@@ -1,43 +1,58 @@
 ---
 # TODO: allow for overriding default ports where possible
-# TODO: trigger the external service when restart is needed
 
 - name: Set node OpenShift facts
   openshift_facts:
-    role: 'node'
+    role: "{{ item.role }}"
+    local_facts: "{{ item.local_facts }}"
+  with_items:
+  - role: common
+    local_facts:
+      hostname: "{{ openshift_hostname | default(none) }}"
+      public_hostname: "{{ openshift_public_hostname | default(none) }}"
+      deployment_type: "{{ openshift_deployment_type }}"
+  - role: node
     local_facts:
+      resources_cpu: "{{ openshift_node_resources_cpu | default(none) }}"
+      resources_memory: "{{ openshift_node_resources_memory | default(none) }}"
+      pod_cidr: "{{ openshift_node_pod_cidr | default(none) }}"
+      labels: "{{ openshift_node_labels | default(none) }}"
+      annotations: "{{ openshift_node_annotations | default(none) }}"
+      registry_url: "{{ oreg_url | default(none) }}"
       debug_level: "{{ openshift_node_debug_level | default(openshift.common.debug_level) }}"
 
-- name: Test if node certs and config exist
-  stat: path={{ item }}
-  failed_when: not result.stat.exists
-  register: result
-  with_items:
-  - "{{ openshift_node_cert_dir }}"
-  - "{{ openshift_node_cert_dir }}/ca.crt"
-  - "{{ openshift_node_cert_dir }}/client.crt"
-  - "{{ openshift_node_cert_dir }}/client.key"
-  - "{{ openshift_node_cert_dir }}/.kubeconfig"
-  - "{{ openshift_node_cert_dir }}/node-config.yaml"
-  - "{{ openshift_node_cert_dir }}/server.crt"
-  - "{{ openshift_node_cert_dir }}/server.key"
-
 - name: Install OpenShift Node package
   yum: pkg=openshift-node state=installed
-  register: install_result
+  register: node_install_result
+
+- name: Install openshift-sdn-ovs
+  yum: pkg=openshift-sdn-ovs state=installed
```

Comment:
Heh, funny. The documentation doesn't match up with the actual module code.  Even odder is that the module default is installed.

I'll update to match with the docs though.